src/rshim_pcie.c

```diff
@@ -861,6 +861,7 @@ static void rshim_pcie_intr(rshim_pcie_t *dev)
       delay = RSHIM_PCIE_RST_START_MIN_DELAY;
     sleep(delay);
     bd->drop_mode = drop_mode;
+    bd->locked_mode = 0;
```

Comment:
no need for this line I think. line 856~863 doesn't really change the drop_mode. It just temporarily enable drop_mode for some clearn-up and changed back.